src/dmd/backend/inliner.d

```diff
@@ -539,8 +548,8 @@ private elem* initializeParamsWithArgs(elem *e, ref int pn, SYMIDX sistart)
                             }
                             assert(e.Eoper == OPvar);
                             elem* e2 = el_copytree(e);
-                            e.EV.Voffset = 0;
-                            e2.EV.Voffset = szs;
+                            e.EV.Voffset += 0;
```

Comment:
This does nothing, but ok.

Reply:
I kinda wanted to emphasize that both should move in tandem. It's more of a "document using the code" kind of thing.